src/ServiceInsight/App_Packages/Particular.Licensing/License.cs

```diff
@@ -41,7 +41,7 @@ public License()
 
         public bool ValidForApplication(string applicationName)
         {
-            return ValidApplications.Contains(applicationName) || ValidApplications.Contains("All");
+            return ValidApplications.Contains(applicationName) || ValidApplications.Contains("All") || applicationName == "NServiceBus";
```

Comment:
It was added to accommodate for older licenses so I think we can just leave as is unless its causing issues for SI?

See:

https://github.com/Particular/Operations.Licensing/pull/28